VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -153,7 +153,7 @@
   min-width: 300px;
   box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
   padding: 12px 16px;
-  z-index: 101;
+  z-index: 999;
```

Comment:
Maybe we need to think about setting a numbering system for this. But this is fine for now.